orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/config/PreconfiguredWebhookProperties.groovy

```diff
@@ -88,6 +88,7 @@ class PreconfiguredWebhookProperties {
     String defaultValue
     String description
     ParameterType type = ParameterType.string
+    int order
```

Comment:
Should this be `int order = 0` or will it default to 0?

Reply:
I've tested and confirm that order returns 0 through Gate if not specified in orca-local.yml.

Reply:
👍